bots/pr/src/main/java/org/openjdk/skara/bots/pr/ApproveCommand.java

```diff
@@ -132,7 +132,7 @@ private List<Issue> getIssues(String issueId, PullRequest pr, List<Comment> allC
             if (issueIds.contains(issue.shortId())) {
                 ret.add(issue);
             } else {
-                reply.println("You can only handle approval request in issues that this pull request solves.");
+                reply.println("You can only handle approval request for issues that this pull request solves.");
```

Comment:
Sorry, I missed this earlier.

```suggestion
                reply.println("You can only handle approval requests for issues that this pull request solves.");
```